academy-2020-piemdb/scripts/createTable.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-console */
+// Create AWS client
+const AWS = require('aws-sdk');
+const fs = require('fs');
+
+AWS.config.update({
+  region: 'localhost',
+  endpoint: 'http://localhost:8000',
+  accessKeyId: 'cpeahq',
+  secretAccessKey: 'y1t6ot',
```

Comment:
Could we use strings that are more clearly just any string to make aws happy (or even remove them completely)